userguide/system.rst

```diff
@@ -735,8 +736,8 @@ Self-Encrypting Drives
 
 Three types of SED devices are supported:
 
-* Legacy interface for older ATA devices (not recommended for
-  security-critical environments)
+* Legacy interface for older ATA devices, which is not recommended for
+  security-critical environments
 
```

Comment:
Change
Legacy interface for older ATA devices, which is not recommended for security-critical environments
to
Legacy interface for older ATA devices. **Not recommended for security-critical environments**
Should be bold or italic.